packages/components/src/List/ListComposition/SortBy/SortBy.component.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Navbar, NavDropdown, Nav, NavItem, MenuItem } from 'react-bootstrap';
+import uuid from 'uuid';
+
+import { useListContext } from '../context';
+
+function SortBy(props) {
+	const { id, options, isDescending, onChange, onOrderChange } = props;
+	const { t } = useListContext();
+
+	// Sort by
+	const selectedOption = props.selected || options[0].key;
+	const onSelect = onChange ? (value, event) => onChange(event, value) : null;
+	const selectedLabel = options.find(option => option.key === selectedOption).label;
+
+	// Sort order
+	const onOrderClick = event => onOrderChange(event, { isDescending: !isDescending });
```

Comment:
I'm asking myself if it will be relevant to implement a default behavior, like an alphabetical one.

Reply:
Or even a default one by type maybe string / number / date. 🤔

Reply:
Yep, but the default/uncontrolled mode range of action and impact is a bit hard to define